platform-sdk/swirlds-common/src/main/java/com/swirlds/common/metrics/MetricConfig.java

```diff
@@ -78,17 +78,17 @@ public abstract sealed class MetricConfig<T extends Metric, C extends MetricConf
             final String unit,
             final String format) {
 
-        this.category = throwArgBlank(category, "category");
-        this.name = throwArgBlank(name, "name");
-        this.description = throwArgBlank(description, "description");
+        this.category = ArgumentUtils.throwArgBlank(category, "category");
```

Comment:
could you please use static imports here?

Reply:
will do next https://github.com/hashgraph/hedera-services/issues/7788